systems_cross_building/cross_building.rst

```diff
@@ -302,6 +302,8 @@ RPI one:
 
 .. code-block:: text
 
+    include(default)
```

Comment:
The thing with this approach is that it relies on the profile to be correctly overwriting and not appending most of the default profile. I think I would add the ``os_build`` and ``arch_build`` explicitly in the profile, not including(default). At least in my experience it is very unusual to have a cross-building profile that is capable of cross-building with the same profile in different OSs, as the cross-compiler paths, etc are typically different in any case.

Reply:
I chose this approach as the file was indeed overwriting everything from the default profile. A nicer fix would be to allow importing only certain settings from a profile, would that be possible?

e.g.:

```
import(default).settings.os_build
# or
import settings.os_build from default
```

Another approach would be to generate a `default` and a `default_host` profile. `default_host`  could be imported in all cross-compilation profiles?

As for hard-coding, the docs would look sub-optimal because the topic is called "Linux/Windows/macOS to Android" - we'd have to split the chapter in three, or add an ugly place-holder :/